src/ert/scheduler/lsf_driver.py

```diff
@@ -87,21 +87,18 @@ def __init__(
         self._retry_sleep_period = 3
 
     async def submit(
-        self, iens: int, executable: str, /, *args: str, cwd: str, name: str = "dummy"
+        self,
+        iens: int,
+        executable: str,
+        /,
+        *args: str,
+        name: str = "dummy",
+        runpath: Optional[str] = None,
```

Comment:
`runpath` refers to `job_runpath`. Just making sure we use it correctly as it is quite overloaded term.

Reply:
hmm, I think runpath is always one of the directories `realization-*/iter-*`, where does it mean anything else?

Reply:
Yes, I double-checked it is `realization-*/iter-*` so it's good :+1: